client/src/store/cards/cards.action.ts

```diff
@@ -1,12 +1,15 @@
 import { Action } from '@ngrx/store';
 import { Card } from 'models';
+import { CdkDragDrop } from '@angular/cdk/drag-drop';
```

Comment:
please specify ordered-imports in tslint 
`        "ordered-imports": [
            true,
            {
                "grouped-imports": true,
                "import-sources-order": "lowercase-last",
                "named-imports-order": "lowercase-last",
                "module-source-path": "full"
            }
        ]`